code/__HELPERS/mobs.dm

```diff
@@ -74,6 +74,10 @@
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_antennae, GLOB.moth_antennae_roundstart_list)
 	if(!GLOB.moth_markings_roundstart_list.len)
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_markings, GLOB.moth_markings_roundstart_list)
+	if(!GLOB.wings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/wings, GLOB.wings_list)
+	if(!GLOB.moth_wings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_wings, GLOB.moth_wings_list)
```

Comment:
What's this all about? Doesn't this exist somewhere else?

Reply:
erroneous duplication when the pr had a merge conflict with #8130

fixed now